lib/libutils/ext/include/bitstring.h

```diff
@@ -144,4 +144,30 @@ typedef	unsigned char bitstr_t;
 	*(value) = _value; \
 } while (0)
 
+#define	bit_ffs_from(name, nbits, startbit, value) do { \
+	register bitstr_t *_name = (name); \
+	register int _byte, _nbits = (nbits); \
+	register int _stopbyte = _bit_byte(_nbits - 1), _value = -1; \
+	register int _startbyte = _bit_byte(startbit); \
+	register int _test_bit_mask = (uint8_t)(0xff << ((startbit) % 8)); \
```

Comment:
With `uint8_t` you add a dependency on `<stdint.h>`, I would probably have used `bitstr_t`  here instead to avoid that, especially since you already have `bitstr_t _lb` below.